packages/cli/src/lib/yargs-cli.ts

```diff
@@ -42,10 +43,13 @@ export function yargsCli<T = unknown>(
     .help()
     .version(false)
     .alias('h', 'help')
+    .check(args => {
+      const persist = args['persist'] as PersistConfig | undefined;
+      return persist == null || validatePersistFormat(persist);
+    })
     .parserConfiguration({
       'strip-dashed': true,
     } satisfies Partial<ParserConfigurationOptions>)
-    .array('persist.format')
```

Comment:
Not sure deleting this is a good idea. It's meant to parse `--persist.format json --persist.format md` and `--persist.format json md` as `['json', 'md']`.

Reply:
The `.array()` function seems to be identical to `type: 'array'` set directly in the options which is already [in place](https://github.com/code-pushup/cli/blob/main/packages/cli/src/lib/implementation/core-config-options.ts#L34-L37).
![image](https://github.com/code-pushup/cli/assets/6306671/6d296f77-ee3c-4a51-bf8a-8fbac15c0f11)

Reply:
Cool, then it was superfluous, and we don't need it :+1: